Neos.Flow/Classes/Configuration/RouteConfigurationProcessor.php

```diff
@@ -216,16 +210,21 @@ protected function buildSubRouteConfigurations(array $routesConfiguration, array
     /**
      * Replaces placeholders in the format <variableName> with the corresponding variable of the specified $variables collection.
      *
-     * @param string $string
+     * @param string|array $value
      * @param array $variables
      * @return string
      */
-    protected function replacePlaceholders($string, array $variables)
+    protected function replacePlaceholders($value, array $variables)
     {
-        foreach ($variables as $variableName => $variableValue) {
-            $string = str_replace('<' . $variableName . '>', $variableValue, $string);
+        if (is_array($value)) {
+            foreach($value as $arrayKey => $arrayValue) {
+                $value[$arrayKey] = $this->replacePlaceholders($arrayValue, $variables);
+            }
+        } elseif (is_string($value)) {
```

Comment:
The `is_string()` check is the actual fix, but it led to wrong results with (nested) array values. So I tweaked this method to work recursively which leads to less code in the lines above as a nice side effect

Reply:
btw, did you know: https://3v4l.org/F49Dq o.O